DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs

```diff
@@ -56,10 +56,12 @@ public override Task CreateAsync(AuditTrailCreateContext context)
                         var formSubmitAction = contentApprovalPart.FormSubmitAction;
                         if(formSubmitAction != null)
                         {
+                            (string button, string action) = formSubmitAction;
+
                             // Publish/Force Publish button
-                            if (formSubmitAction.Item1.Contains(Constants.SubmitPublishKey))
+                            if (button.Contains(Constants.SubmitPublishKey))
                             {
-                                switch (formSubmitAction.Item2)
+                                switch (button)
```

Comment:
Item2 is supposed to be action not button?

Reply:
Done.